YACReader/main_window_viewer.cpp

```diff
@@ -1039,6 +1039,11 @@ void MainWindowViewer::disableActions()
 
 void MainWindowViewer::keyPressEvent(QKeyEvent *event)
 {
+    if (1) {
+        QMainWindow::keyPressEvent(event);
+        return;
+    }
+
```

Comment:
This is probably an old workaround to fix problems with multi key shortcuts, i.e. "CTRL+F" for fullscreen and other possible stuff. If you can make this work with all event modifier keys below there is no need for a custom keyPressEvent function at all, so the best approach is to simply remove it.

Reply:
I have verified that the standard `QAction` shortcut mechanism supports all 4 modifier keys that were handled in `keyPressEvent()`s.